hack/remove-members.go

```diff
@@ -0,0 +1,184 @@
+package main
+
+import (
+	"bufio"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"regexp"
+	"sort"
+	"strings"
+)
+
+var dryrun bool
+var repoRoot string
+
+func main() {
+	flag.StringVar(&repoRoot, "root", ".", "Root of the repository")
```

Comment:
Instead of allowing to specify the repoRoot, we could probably take the config as a flag. For example, if we want to selectively remove only members in `kubernetes`, we should be able to specify that